slave/pistoncloud/fabfile.py

```diff
@@ -0,0 +1,227 @@
+# Copyright Hybrid Logic Ltd.
+"""
+Configuration for a buildslave to run on PistonCloud
+
+.. warning::
+    This points at the staging buildserver by default.
+"""
+
+from fabric.api import sudo, task, env, execute, put, run
+from fabric.context_managers import shell_env
+from twisted.python.filepath import FilePath
+from StringIO import StringIO
+import yaml
+
+# http://stackoverflow.com/a/9685171
+env.use_ssh_config = True
+
+BUILDSLAVE_NAME = "clusterhq_flocker_buildslave"
+# Be careful here! If our script has bugs we don't want to accidentally
+# modify VMs or resources of another more important tenant
```

Comment:
In order to be careful I must know:
* How this is used
* How I could modify resources of another tenant (by using the same name?)